File: C1.UWP.FlexChart/CS/GestureChartSample/GestureModeConverter.cs
using C1.Xaml.Chart.Interaction;
using System;
using Windows.UI.Xaml.Data;

namespace GestureChartSample
{
    class GestureModeToStringConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            return value.ToString();
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            return (GestureMode)Enum.Parse(typeof(GestureMode), value.ToString());
        }
    }
}

File: C1.UWP.FlexChart/VB/GestureChartSample/GestureModeConverter.vb
Imports Windows.UI.Xaml.Data
Imports System
Imports C1.Xaml.Chart.Interaction

Class GestureModeToStringConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Return value.ToString()
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Return CType([Enum].Parse(GetType(GestureMode), value.ToString()), GestureMode)
    End Function
End Class